.env

```diff
@@ -0,0 +1,22 @@
+# Django Secret Key
+APP_SECRET_KEY = change_me
+
+# Making Python aware of where to start
+PYTHONPATH=/home/GraphQLApp
+
+# Data dir
+DATA_DIR=/path/to/the/data/
```

Comment:
So, do we need a storage directory for the application?

Reply:
As for now, I think we don't need it